Strategies/Account.py

```diff
@@ -0,0 +1,51 @@
+class Account():
+    def __init__(self, starting_balance=0, starting_shares=0):
+        self.balance = float(starting_balance)
+        self.shares = float(starting_shares)
+        
+    def get_balance(self):
+        return self.balance
+    
+    def get_shares(self):
+        return self.shares
+    
+    def set_balance(self, balance):
+        self.balance = balance
+        
+    def set_shares(self, shares):
+        self.shares = shares
+    
+    def increase_balance(self, increase):
+        self.balance += increase
+        
+    def increase_shares(self, increase):
+        self.shares += increase
+    
+    def sell(self, share_price, shares_to_sell=None):
+        if shares_to_sell:
+            if shares_to_sell > self.shares:
+                raise ValueError("Attempted to sell more shares than you own.")
+                quit
```

Comment:
Have you tried without the quit? Are you planning on trying to catch the ValueError when buying or selling?